src/app/modules/notification/services/notification.service.ts

```diff
@@ -0,0 +1,111 @@
+import { Injectable } from '@angular/core';
+
+import { Notification } from '../@type/notification';
+import { NotificationOptions } from '../@type/notification-options';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class NotificationService {
+
+  /** Default notifiactions options */
+  options: NotificationOptions = {
+    isDismissable: true,
+    isSelfClosing: true,
+    duration: 5
+  };
+
+  /** Notifications list */
+  private notifications: Notification[] = [];
```

Comment:
it's the same for every attribute, you don't really need private word and getters :)

Reply:
I thought it was better not to be able to access the attributes directly...
So when is it relevant to have getters?

Reply:
:+1: